### ISSUE TITLE
GA4: Appointments Team - Custom Events (Implementation)

### ISSUE THREAD

**Issue by johnny-jesensky-adhoc** [department-of-veterans-affairs/va.gov-team#76912]
### Description
We have the list of custom events we will need to start mapping out for the Appointments Team (VAOS). This task is to set up the tracking for these events within GA4.

- **Custom Events:** https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/analytics/%5Bdraft%5Dvaos-analytics-definitions.md

_Note that testing of GA4 events will need to be done with assistance from the Appointments Team._

---

### Tasks
- [x]  Map identified custom events to GA4 fields/schema
- [ ]  Add each component event to GTM
- [ ]  Need approval from Appointments Team as part of this

### Acceptance Criteria
- [ ] Tracking document has been created and delivered to Jeff Roof and Appointments Team.

**Comment by johnny-jesensky-adhoc** [comment]
With me being OOO for the last week and Jamie being out sick today on my first day back, I haven't been able to confirm if this is _Done_,. I am going to keep this open until I have validated this with Jamie -- hopefully tomorrow.

FYI @jestutt, @davidconlon, @jwoodman5

**Comment by jestutt** [comment]
@johnny-jesensky-adhoc 
Custom events mapping spreadsheet with questions: [vaos_custom-events_v1.xlsx](https://github.com/department-of-veterans-affairs/va.gov-team/files/15221272/vaos_custom-events_v1.xlsx)

**Comment by johnny-jesensky-adhoc** [comment]
We've made some good progress on this, as outlined in Jamie's file above, (screenshot of file), though we hit a blocker with some questions we had for their team. We weren't able to meet with the team until yesterday, (5/7), due to our main point of contact with the team being out on medical leave. 

While we cleared some things up with some other resources on their team, we ultimately weren't able to get through all of this task in the current sprint. We plan to finish implementing these custom events next sprint, with further refinement outline in a separate ticket.
<img width="593" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/147662408/a54f277c-9089-4919-be62-0cebc122e4bc">


FYI @jestutt, @davidconlon, @jwoodman5
